shared/ui-components/package.json

```diff
@@ -26,6 +26,7 @@
     "@types/enzyme-adapter-react-16": "^1.0.6",
     "@types/jest": "^25.2.2",
     "@types/jwt-decode": "^2.2.1",
+    "@types/react-input-mask": "^2.0.5",
```

Comment:
Is this related to this PR? I didn't see any other changes to `ui-components`...

Reply:
No, but I think we merged it in from a different PR in the last couple days. It would be good to remove it here (and from the yarn.lock) when cleaning up the merge conflicts.

Reply:
I removed it, although it is still being displayed in Github's PR diff view..